corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -655,10 +658,18 @@ hqDefine("cloudcare/js/formplayer/menus/views", function () {
         templateContext: function () {
             const paginateItems = formplayerUtils.paginateOptions(this.options.currentPage, this.options.pageCount);
             const casesPerPage = parseInt($.cookie("cases-per-page-limit")) || 10;
+            const boldSortedCharIcon = (header) => {
+                const header_words = header.trim().split(' ');
+                const lastChar = header_words.pop();
+
+                return lastChar === "Λ" || lastChar === "V"
+                  ? `${header_words.join(' ')} <b>${lastChar}</b>`
+                  : header;
+              };
```

Comment:
The headers for regular case list are already bolded, so visually it won't have an affect. But in the event the headers are changed to not be bolded, I think it would still be good to bold the "sort arrows". So I think we should keep it here.

But I will note this change in the PR description